tests/actions/visual-recognition-v3/service-bind.test.js

```diff
@@ -0,0 +1,73 @@
+const assert = require('assert');
+const nock = require('nock');
+const extend = require('extend');
+const omit = require('object.omit');
+const path = require('path');
+const fs = require('fs');
+const openwhisk = require('openwhisk');
+const { auth, describe } = require('../../resources/auth-helper');
+const { adapt, negativeHandler } = require('../../resources/test-helper');
+let listClassifiers = require('../../../packages/visual-recognition-v3/actions/list-classifiers');
+
+const IAM_HOST = 'https://iam.bluemix.net:443';
```

Comment:
This file has unused variables and global variables.

What are you testing here? 
Can you add a test for when the credentials are not in `__bx_creds` and one for when it's a cf service